demos/serverless/template.yaml

```diff
@@ -7,6 +7,15 @@ Parameters:
     Default: false
     Type: String
     AllowedValues: [true, false]
+  Region:
+    Description: The AWS SDK Chime REGION
+    Default: "us-east-1"
+    Type: String
+  UseChimeMeetingsSDK:
+    Description: Use Regional to switch between Chime and ChimeSDKMeetings Client
```

Comment:
Should this be: "Use boolean to switch between Chime and ChimeSDKMeetings Client" ?